### PR TITLE
[zh-cn] Update performance/using dns prefetch

### PR DESCRIPTION
**Pull request by jasonren0403**

### CODE REVIEW

File: files/zh-cn/web/performance/dns-prefetch/index.md [hunk 1]
```diff
@@ -1,68 +1,72 @@
...
 
-**`DNS-prefetch`** (**DNS 预获取**) 是尝试在请求资源之前解析域名。这可能是后面要加载的文件，也可能是用户尝试打开的链接目标。
+{{QuickLinksWithSubPages("Web/Performance")}}
 
-## 为什么要使用 dns-prefetch?
+**`DNS-prefetch`** 尝试在请求资源之前解析域名。这可能是后面要加载的文件，也可能是用户尝试打开的链接目标。
+
+## 为什么要使用 dns-prefetch？
 
 当浏览器从（第三方）服务器请求资源时，必须先将该[跨域](/zh-CN/docs/Web/HTTP/CORS)域名解析为 IP 地址，然后浏览器才能发出请求。此过程称为 DNS 解析。DNS 缓存可以帮助减少此延迟，而 DNS 解析可以导致请求增加明显的延迟。对于打开了与许多第三方的连接的网站，此延迟可能会大大降低加载性能。
```
Comment (jasonren0403) [inline files/zh-cn/web/performance/dns-prefetch/index.md:12]:
```suggestion
当浏览器从（第三方）服务器请求资源时，必须先将该[跨源](/zh-CN/docs/Web/HTTP/CORS)域名解析为 IP 地址，然后浏览器才能发出请求。此过程称为 DNS 解析。DNS 缓存可以帮助减少此延迟，而 DNS 解析可以导致请求增加明显的延迟。对于打开了与许多第三方的连接的网站，此延迟可能会大大降低加载性能。
```

File: files/zh-cn/web/performance/dns-prefetch/index.md [hunk 2]
```diff
@@ -1,68 +1,72 @@
...
   </head>
   <body>
-    <!-- your page content -->
+    <!-- 你的页面内容 -->
   </body>
 </html>
 ```
 
-每当站点引用跨域域上的资源时，都应在[\<head>元素](/zh-CN/docs/Web/HTML/Element/head)中放置 dns-prefetch 提示，但是要记住一些注意事项。
+每当站点引用跨域资源时，都应在 [`<head>` 元素](/zh-CN/docs/Web/HTML/Element/head)中放置 `dns-prefetch` 提示，但是要记住一些注意事项。
```
Comment (yin1999) [inline files/zh-cn/web/performance/dns-prefetch/index.md:38]: +1

File: files/zh-cn/web/performance/dns-prefetch/index.md [hunk 3]
```diff
@@ -1,68 +1,72 @@
...
 
 ## 最佳实践
 
 请记住以下三点：
 
-首先，`dns-prefetch` 仅对[跨域](/zh-CN/docs/Web/HTTP/CORS)域上的 DNS 查找有效，因此请避免使用它来指向您的站点或域。这是因为，到浏览器看到提示时，您站点域背后的 IP 已经被解析。
+**首先**，`dns-prefetch` 仅对[跨域](/zh-CN/docs/Web/HTTP/CORS)域上的 DNS 查找有效，因此请避免使用它来指向你的站点或域。这是因为，到浏览器看到提示时，你的站点背后的 IP 已经被解析了。
```
Comment (yin1999) [inline files/zh-cn/web/performance/dns-prefetch/index.md:44]: +1

File: files/zh-cn/web/performance/dns-prefetch/index.md [hunk 4]
```diff
@@ -1,68 +1,72 @@
...
 
 ```html
-<link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
-<link rel="dns-prefetch" href="https://fonts.gstatic.com/">
+<link rel="preconnect" href="https://fonts.googleapis.com/" crossorigin />
+<link rel="dns-prefetch" href="https://fonts.googleapis.com/" />
 ```
 
-> **备注：** 如果页面需要建立与许多第三方域的连接，则将它们预先连接会适得其反。 `preconnect` 提示最好仅用于最关键的连接。对于其他的，只需使用 `<link rel="dns-prefetch">` 即可节省第一步的时间-DNS 查找。
+> **备注：** 如果页面需要建立与许多第三方域的连接，则将它们预先连接会适得其反。`preconnect` 提示最好仅用于最关键的连接。对于其他的连接，只需使用 `<link rel="dns-prefetch">` 即可节省第一步的时间——DNS 查找。
```
Comment (yin1999) [inline files/zh-cn/web/performance/dns-prefetch/index.md:59]:
```suggestion
> **备注：** 如果页面需要建立与许多第三方域的连接，则将它们预先连接会适得其反。`preconnect` 提示最好仅用于最关键的连接。对于其他的连接，只需使用 `<link rel="dns-prefetch">` 即可节省第一步——DNS 查询——的时间。
```

**Review by yin1999 — APPROVED** [review]
two occurences left. And one more suggestion. Otherwise looks good :)

File: files/zh-cn/web/performance/dns-prefetch/index.md (hunk 3 above)
Comment (jasonren0403) [reply files/zh-cn/web/performance/dns-prefetch/index.md:44]:
```suggestion
**首先**，`dns-prefetch` 仅对[跨源](/zh-CN/docs/Web/HTTP/CORS)域上的 DNS 查找有效，因此请避免使用它来指向你的站点或域。这是因为，到浏览器看到提示时，你的站点背后的 IP 已经被解析了。
```

File: files/zh-cn/web/performance/dns-prefetch/index.md (hunk 2 above)
Comment (jasonren0403) [reply files/zh-cn/web/performance/dns-prefetch/index.md:38]:
```suggestion
每当站点引用跨源资源时，都应在 [`<head>` 元素](/zh-CN/docs/Web/HTML/Element/head)中放置 `dns-prefetch` 提示，但是要记住一些注意事项。
```

**Pull request merged**